open_alchemy/build/__init__.py

```diff
@@ -246,3 +249,89 @@ def execute(*, spec: typing.Any, name: str, path: str) -> None:
         spec=spec_str,
         init=init,
     )
+
+    # Build a distribution package if necessary.
+    if format_:
+        pkg_dir = pathlib.Path(path) / name
+        build_dist(str(pkg_dir), format_)
+
+
+def build_dist(path: str, format_: str) -> None:
+    """
+    Build a distibution package.
+
+    Args:
+        path: The package directory.
+        format: The package format to build ("sdist", "wheel" or "all").
+    """
+    formats = {"sdist", "wheel", "all"}
+    fmt = format_.lower()
+    if fmt not in formats:
+        raise ValueError(
+            "{} is not a valid format (use one of: {})".format(
+                format_, ", ".join(formats)
+            )
+        )
+    if fmt == "sdist":
+        build_sdist(path)
+    elif fmt == "wheel":
+        build_wheel(path)
+    else:
+        build_sdist(path)
+        build_wheel(path)
+
+
+def build_sdist(path: str) -> None:
+    """
+    Build a .tar.gz source distribution and place it in a "dist" folder.
+
+    The "dist" folder is located at the root of the package.
+
+    Args:
+        path: The package directory.
+    """
+    run("python setup.py sdist", path)
```

Comment:
I haven't encountered this before, does this execute a shell command? If so, how can you be sure that the `python` command does what you think it will do? For example, it could point to Python 2.7 on certain systems. Is there a Python API that can produce these?